### PR TITLE
feat : iOS 연동 앱스킴 실행 추가 [PCM-354]

### PR DESCRIPTION
**Pull request by kjh5833**
https://chai-finance.slack.com/archives/C01FJSM3XC5/p1637048729383000?thread_ts=1637042706.377500&cid=C01FJSM3XC5

### CODE REVIEW

File: 인증결제/README.md [hunk 1]
```diff
@@ -302,9 +256,53 @@ public class MyViewClient extends WebViewClient {
...
+| kakaotalk | 카카오페이 |
+| kb-acp | 국민카드-앱카드 |
+| kb-auth | 국민 |
+| kftc-bankpay | 계좌이체 |
+| lguthepay-xpay | 페이나우 |
+| liivbank | 국민 Liiv M(리브모바일) |
+| lmslpay | 롯데 L.pay 앱 |
+| lotteappcard | 롯데카드-앱카드 |
+| lottesmartpay | 롯데카드-모바일결제 |
+| Lpayapp | (구)롯데 L.pay 앱 |
```
Comment (anymate98) [inline 인증결제/README.md:290]: 소문자로 바꿔주세요~

File: 인증결제/README.md [hunk 2]
```diff
@@ -302,9 +256,53 @@ public class MyViewClient extends WebViewClient {
...
+| hanamopmoasign | 하나카드-공인인증 앱 |
+| hanawalletmembers | 하나카드-하나멤버스 월렛 |
+| hdcardappcardansimclick | 현대카드-앱카드 |
+| hyundaicardappcardid | 현대카드 |
+| ispmobile | ISP모바일 |
+| itms-apps | 앱스토어 |
+| kakaotalk | 카카오페이 |
+| kb-acp | 국민카드-앱카드 |
+| kb-auth | 국민 |
+| kftc-bankpay | 계좌이체 |
```
Comment (anymate98) [inline 인증결제/README.md:284]: 뱅크페이 앱이라고 하는게 어떨까요?

File: 인증결제/README.md [hunk 3]
```diff
@@ -314,6 +312,24 @@ iOS 보안 정책상 외부 호출될 URL scheme을 `info.plist` 파일의 `LSAp
 </array>
 ```
 
+또한, URL 변경시 웹뷰내 페이지 전환을 하지 않고, 앱스킴을 실행시키려면 아래 코드를 구현하셔야 합니다.
```
Comment (anymate98) [inline 인증결제/README.md:315]: "url 변경 과정에서 앱 url을 감지했을 때"라고 표현하는 것이 어떨까요?

**Review by yisak9101 — APPROVED** [review]
수고하셨습니다!

File: 인증결제/README.md (hunk 3 above)
Comment (kjh5833) [reply 인증결제/README.md]: https://github.com/iamport/iamport-manual/pull/49/commits/eebdb2e4a07f8c0136301b570a0b1cae1bec345e

File: 인증결제/README.md (hunk 2 above)
Comment (kjh5833) [reply 인증결제/README.md]: https://github.com/iamport/iamport-manual/pull/49/commits/eebdb2e4a07f8c0136301b570a0b1cae1bec345e

File: 인증결제/README.md (hunk 1 above)
Comment (kjh5833) [reply 인증결제/README.md]: https://github.com/iamport/iamport-manual/pull/49/commits/eebdb2e4a07f8c0136301b570a0b1cae1bec345e

**Pull request merged**